src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsHearingRecordingCaseData.java

```diff
@@ -16,5 +17,20 @@
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class SscsHearingRecordingCaseData {
     private DynamicList selectHearingDetails;
+    private DynamicList requestableHearingDetails;
+    private List<HearingRecordingRequest> requestedHearings;
+    private List<HearingRecordingRequest> releasedHearings;
+    private String requestedHearingsTextList;
+    private String releasedHearingsTextList;
+    private YesNo hearingRecordingRequestOutstanding;
+
+    @JsonProperty("hearingRecordings")
```

Comment:
Why do you need the Json Property here?

Reply:
don't think we do, copied it in from Divya's PR